cpan_deps_graph.pl

```diff
@@ -117,32 +131,33 @@
   return \%all_deps;
 };
 
-helper dist_dep_tree => sub ($c, $dist, $phases, $relationships, $perl_version) {
+helper dist_dep_tree => sub ($c, $dist, $phases, $relationships, $perl_version, $dist_version = undef) {
   my %seen;
   my %deps;
-  my @to_check = $dist;
-  while (defined(my $dist = shift @to_check)) {
+  my @to_check = [ $dist, $dist_version ]; # only pass on version for initial
```

Comment:
I'd prefer this meta structure was a hashref than an arrayref for clarity

Reply:
Done